clients/js/src/chains/sui/utils.ts

```diff
@@ -38,6 +38,7 @@ export const executeTransactionBlock = async (
   const consoleWarnTemp = console.warn;
   console.warn = () => {};
 
+  transactionBlock.setGasBudget(100_000_000);
```

Comment:
a comment pointing to an explorer link for a successful transaction would be nice to give some context to this number

Reply:
Yeah, when I made this change I was actually copying what the sdk already did at that point but I think we should have proper gas estimation instead.